hw/xbox/xid.c

```diff
@@ -360,14 +376,28 @@ static void usb_xid_handle_control(USBDevice *dev, USBPacket *p,
             memcpy(data, s->xid_desc, s->xid_desc->bLength);
             p->actual_length = s->xid_desc->bLength;
         } else {
+            p->status = USB_RET_STALL;
             assert(false);
         }
         break;
     case VendorInterfaceRequest | XID_GET_CAPABILITIES:
         DPRINTF("xid XID_GET_CAPABILITIES 0x%x\n", value);
-        /* FIXME: ! */
-        p->status = USB_RET_STALL;
-        //assert(false);
+        if (value == 0x0100) {
```

Comment:
nit: might want to use constants instead of the values here for readability (but I won't block merge for it)

Reply:
I agree and I did consider it.

However, I wasn't sure which names to pick + I'm not even sure how these constants really work. Other XID projects I checked didn't seem to use or name them either.
These constants were also un-named in the existing code.

I consider this PR to be mostly a fixup for wrong behaviour and not the vehicle for design or code quality changes. Another reason was that I wanted to avoid larger changes if we have to repeat them in xid-sdl soon.
So I think this should be handled when making XID design decisions when updating or merging with xid-sdl.